internal/cli/plugin/install.go

```diff
@@ -35,8 +35,8 @@ type InstallOpts struct {
 }
 
 func (opts *InstallOpts) checkForDuplicatePlugins() error {
-	for _, plugin := range opts.plugins {
-		if plugin.Github != nil && plugin.Github.Equals(opts.githubRelease.owner, opts.githubRelease.name) {
+	for _, p := range opts.plugins {
```

Comment:
Why did you rename this? `plugin` is more readable than `p`

Reply:
I got an error from the linter saying that I cannot use "plugin" because it is the name of a package